docs/docs/writing-stories.mdx

```diff
@@ -399,7 +399,7 @@ In some cases, such as single-turn interjections, you can handle context-switchi
 using [rules](./rules.mdx) instead of through [stories](./stories.mdx).
 
 Consider this conversation scenario:
-<Chat caption="Single-turn interjection">
+<Chat caption="A single-turn interjection">
```

Comment:
You might get flaky results from MDX if you don't leave empty lines between markdown and JSX instructions